apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/DiagnosisUtilV2.java

```diff
@@ -139,7 +139,7 @@ static CodeableConcept toCodeableConcept(Diagnosis diag) {
      * must be added. A coding with the ICD-10 system URL will still be present for backwards compatibility.
      * See JIRA ticket: https://jira.cms.gov/browse/BFD-1894
      */
-    if (system == IcdCode.CODING_SYSTEM_ICD_10) {
+    if (system.equals(IcdCode.CODING_SYSTEM_ICD_10)) {
```

Comment:
Was there a test for this that needed to be adjusted? Or is there an existing test that happened to pass due to using the same static string instance?

Reply:
No test needed. Test was using static string instance.

Reply:
Perhaps a good reason to _not_ use the static instance in the future; it missed this bug due to that.